applications/DEMApplication/python_scripts/main_script.py

```diff
@@ -32,8 +32,11 @@ def GetParametersFileName(self):
 
     def GetInputParameters(self):
```

Comment:
this seems to be a duplicate, maybe you can refactor

Reply:
The same goes with this one. The new DEM_analysis_stage.py will be used instead of the main_script.py.
We are currently in the process of refactoring the code.